cmd/kubeless/function.go

```diff
@@ -192,6 +192,10 @@ func getFunctionDescription(funcName, ns, handler, funcContent, deps, runtime, t
 			Deps:     deps,
 			Topic:    topic,
 			Schedule: schedule,
+			Service: spec.ServiceSpec{
```

Comment:
in case the user is updating the file these new flags may be unspecified. For not overwriting the previous behavior you need to set the default the `defaultFunction` (just like with the other parameters)